### PR TITLE
Create custom managed properties without special characters

### PR DESCRIPTION
**Pull request by zerooneday**
Custom managed property name with special character is not a good choice.

It is possible to create custom managed properties with special characters (e.g. hyphen, dot, underscore etc.)  Since many of those characters act as operator characters in the query syntax, e.g. hyphen which means negation, such property names have to be enclosed in double quotes when used in queries.  Many tools and applications that create queries don’t handle this correctly, and therefore it is strongly recommended to use only letters and digits in managed property names.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/manage-search-schema.md [hunk 1]
```diff
@@ -127,6 +127,9 @@ For more info about Delve, see [Office Delve for Office 365 admins](https://go.m
 > Not all options are available in SharePoint in Microsoft 365.  Refinable Managed Properties cannot be created. Instead use the existing Refinable Managed Properties, e.g. RefinableString00 or RefinableInt00. Searching "Refinable" will show all of the usable Refinable Managed Properties available.
 
 In SharePoint in Microsoft 365, when you create a new managed property, it will have some limitations. For example, the property can only be of type **Text** or **Yes/No**, and it can't be refinable or sortable.
+
+> [!IMPORTANT]
+> Strongly recommended to use only letters and digits in managed property names. Although it is possible to create custom managed properties with special characters (e.g. hyphen, dot, underscore etc.), many of those characters act as operator characters in the query syntax, e.g. hyphen which means negation, such property names have to be enclosed in double quotes when used in queries.  Many tools and applications that create queries don’t handle this correctly, and therefore it is better to use only letters and digits in managed property names.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/manage-search-schema.md:132]:
```suggestion
> We strongly recommend using only letters and digits in managed property names. Although it is possible to create custom managed properties with special characters (such as hyphens, dots, and underscores), many of those characters act as operator characters in the query syntax, for example, a hyphen which means negation. Such property names have to be enclosed in double quotation marks when used in queries.  Many tools and applications that create queries don’t handle this correctly, and therefore it is better to use only letters and digits in managed property names.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**